src/Installer/redist-installer/targets/BundledManifests.targets

```diff
@@ -90,15 +91,37 @@
 
   <Target Name="LayoutManifests"
           DependsOnTargets="SetupBundledComponents">
+
+    <!-- Manifest sources that exist in-tree -->
+    <ItemGroup>
+      <SourceManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\*"
+                             Condition="'%(BundledManifests.IncludedInSource)' == 'true' and Exists('%(BundledManifests.RestoredNupkgContentPath)\data')"
+                             DestinationPath="%(BundledManifests.FeatureBand)/$([MSBuild]::ValueOrDefault('%(Identity)', '').ToLower())/%(BundledManifests.Version)"
+                             RestoredNupkgContentPath="%(RestoredNupkgContentPath)"
+                             WorkloadManifestId="%(Identity)" />
+
+      <SourceManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\localize\*"
+                             Condition="'%(BundledManifests.IncludedInSource)' == 'true' and Exists('%(RestoredNupkgContentPath)\data\localize')"
+                             DestinationPath="%(BundledManifests.FeatureBand)/$([MSBuild]::ValueOrDefault('%(Identity)', '').ToLower())/%(BundledManifests.Version)/localize" />
+    </ItemGroup>
+
+    <!-- Do not copy for source build, should already be in-tree -->
+    <Copy Condition="'$(DotNetBuildSourceOnly)' != 'true'"
+          SourceFiles="@(SourceManifestContent)"
+          DestinationFolder="sdk-manifests/%(DestinationPath)"
+          SkipUnchangedFiles="true" />
```

Comment:
This is for convenience of someone updating the manifests (likely me). We could put it behind a property like `-p:UpdateManifestSources=true`, if we are concerned about this running.